go-packages/meep-gis-asset-mgr/asset-mgr.go

```diff
@@ -245,44 +252,79 @@ func (pc *Connector) CreateDb(name string) (err error) {
 	return nil
 }
 
-func (pc *Connector) CreateTables() (err error) {
-	_, err = pc.db.Exec("CREATE EXTENSION IF NOT EXISTS postgis")
+// DestroyDb -- Destroy DB with provided name
+func (am *AssetMgr) DestroyDb(name string) (err error) {
+	_, err = am.db.Exec("DROP DATABASE " + name)
+	if err != nil {
+		log.Error(err.Error())
+		return err
+	}
+
+	log.Info("Destroyed database: " + name)
+	return nil
+}
+
+func (am *AssetMgr) CreateTables() (err error) {
+	_, err = am.db.Exec("CREATE EXTENSION IF NOT EXISTS postgis")
 	if err != nil {
 		log.Error(err.Error())
 		return err
 	}
 
 	// UE Table
-	_, err = pc.db.Exec(`CREATE TABLE ` + UeTable + ` (
-		id 				varchar(36) 			NOT NULL PRIMARY KEY,
-		name 			varchar(100) 			NOT NULL UNIQUE,
-		position		geometry(POINT,4326)	NOT NULL,
-		path 			geometry(LINESTRING,4326),
-		path_mode       varchar(20)           	NOT NULL DEFAULT 'LOOP',
-		path_velocity   decimal(10,3)         	NOT NULL DEFAULT '0.000',
-		path_length     decimal(10,3)         	NOT NULL DEFAULT '0.000',
-		path_increment  decimal(10,6)         	NOT NULL DEFAULT '0.000000',
-		path_fraction   decimal(10,6)         	NOT NULL DEFAULT '0.000000',
-		poa				varchar(100)			NOT NULL DEFAULT '',
-		poa_distance    decimal(10,3)         	NOT NULL DEFAULT '0.000',
-		poa_in_range	varchar(100)[]			NOT NULL DEFAULT array[]::varchar[],
-		poa_type_prio	varchar(20)[]			NOT NULL DEFAULT array[]::varchar[],
-		connected		boolean					NOT NULL DEFAULT 'false',
-		start_time		timestamptz 			NOT NULL DEFAULT now()
+	_, err = am.db.Exec(`CREATE TABLE ` + UeTable + ` (
+		id              varchar(36)             NOT NULL,
+		name            varchar(100)            NOT NULL UNIQUE,
+		position        geometry(POINT,4326)    NOT NULL,
+		path            geometry(LINESTRING,4326),
+		path_mode       varchar(20)             NOT NULL DEFAULT 'LOOP',
+		path_velocity   decimal(10,3)           NOT NULL DEFAULT '0.000',
+		path_length     decimal(10,3)           NOT NULL DEFAULT '0.000',
+		path_increment  decimal(10,6)           NOT NULL DEFAULT '0.000000',
+		path_fraction   decimal(10,6)           NOT NULL DEFAULT '0.000000',
+		poa             varchar(100)            NOT NULL DEFAULT '',
+		poa_distance    decimal(10,3)           NOT NULL DEFAULT '0.000',
+		poa_in_range    varchar(100)[]          NOT NULL DEFAULT array[]::varchar[],
+		poa_type_prio   varchar(20)[]           NOT NULL DEFAULT array[]::varchar[],
+		connected       boolean                 NOT NULL DEFAULT 'false',
+		start_time      timestamptz             NOT NULL DEFAULT now(),
+		PRIMARY KEY (id)
 	)`)
 	if err != nil {
 		log.Error(err.Error())
 		return err
 	}
 	log.Info("Created UE table: ", UeTable)
 
+	// UE Measurements Table
```

Comment:
We should leverage the power of relational database by linking UE Ids in the different tables.
This would allow to get information from multiple tables in a single SQL operation.

Reply:
UeMeasurementTable does have a relation with the UE table; it uses the UE name as a foreign key for UE measurments.